### PR TITLE
renamed the Features folder and corrected the pathes

### PR DESCRIPTION
**Pull request by bubumariia**

### CODE REVIEW

**Comment by bubumariia** [comment]
localhost_3000_ (3)

**Comment by bubumariia** [comment]
it looks fine and no errors on console

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use client';
 
 import { Footer } from '../components/Footer/Footer';
-import { HeaderPage } from '../Features/header';
+import { HeaderPage } from '../features/header';
 import { PromoCard } from '../components/PromoCard';
```
Comment (ain-odin) [inline src/app/page.tsx:5]: It seems the folder name is still the same
Screenshot 2023-09-13 at 09 55 13
This is why the checks have not been successful.

Comment (Alex-2kZharkov) [reply src/app/page.tsx:5]: yes, you are right Ainura.

@bubumariia you need to rename the folder itself. Click on the folder using the right button of your mouse, look for the `Rename` command if you are on the vs code (or google other ways to do it), For now, you've just renamed the occurrences of that folder, not the folder itself

**Review by Alex-2kZharkov — CHANGES_REQUESTED** [review]
waiting for the changes

**Comment by bubumariia** [comment]
dccab980-50cc-4ef3-b52e-8c7a03157533

Actually, I have renamed it, I don't know why it is happening. Locally, as you can see it is lowercase.

**Comment by bubumariia** [comment]
Let me find out what causes it

**Pull request closed without merging**